src/DrawerScreenNavigators.js

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
```

Comment:
the file name is wrong. they're stacks. also it should be agnostic cause it could be anything. call it 'levelNavigators.js'